src/lib/notifier.js

```diff
@@ -83,80 +59,30 @@ class Notifier {
   }
 
   getLineWithNumber(log) {
-    let lineNum = this.lineIndex++;
-    return '(' + lineNum + ') ' + log;
+    const lineNum = this.lineIndex++;
+    return `(${lineNum}) ${log}`;
   }
 
   notify(log) {
-    let { formatter, color, preNotify } = this.options;
-    let output = formatter.formatLog(log, { color: color });
-
-    if (color) {
-      output = formatter.color(output);
-    }
+    const { formatter, color, preNotify } = this.options;
+    const output = formatter.formatLog(log, { color });
 
-    if (isFunction(preNotify)) {
-      preNotify(log);
-    }
+    if (isFunction(preNotify)) preNotify(log);
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
    if (isFunction(preNotify)) {preNotify(log);}
```